lib/cn/csched_sp3.c

```diff
@@ -834,7 +834,7 @@ sp3_dirty_node_locked(struct sp3 *sp, struct cn_tree_node *tn)
          * in FIFO order, retaining its current position if it's already on
          * the list.  List order is otherwise managed by sp3_check_roots().
          */
-        if (nkvsets >= sp->thresh.rspill_runlen_min && jobs < jobs_max) {
+        if (nkvsets >= 1 && jobs < jobs_max) {
```

Comment:
This allows us to now zspill kvsets ASAP rather than letting them build up.  Just so long as there isn't an ongoing rspill.  With time-series load the root node is almost always empty!